kolibri_explore_plugin/assets/src/kolibriApi.js

```diff
@@ -234,6 +238,22 @@ class KolibriApi {
   get defaultHideUnavailable() {
     return DEFAULT_HIDE_UNAVAILABLE;
   }
+
+  translate(nameSpace, defaultMessages, messageId, args) {
+    // Create the translator and add it to the cache only if needed:
+    let translator;
+    // FIXME this would be more readable by using the nullish coalescing
+    // assignment (??=) operator, but current linter is not happy:
+    // this.translators[nameSpace] ??= createTranslator(nameSpace, defaultMessages);
+    // const translator = this.translators[nameSpace];
```

Comment:
Has the linter regressed? I remember using es6 operators like this in the past.

Reply:
I don't remember changing the linter at all, but we did run into a problem with that operator in https://github.com/endlessm/kolibri-explore-plugin/pull/693.

Reply:
I changed the babel configuration in https://github.com/endlessm/kolibri-explore-plugin/pull/744 and verified that it accepts nullish coalescing operators now. But I didn’t explicitly test the linter; perhaps it could be subject to the same problem? Does it load the babel config?

Reply:
https://mastodon.social/@gknauss/111046912763765216